python/lsst/ts/standardscripts/base_take_image.py

```diff
@@ -153,6 +195,30 @@ def set_metadata(self, metadata):
             * nimages
         )
 
+    def set_additional_nextvisit_metadata(self, metadata):
+        metadata.nimages = len(self.config.exp_times)
+        metadata.coordinateSystem = MetadataCoordSys.ICRS
+        if (self.ra is not None) and (self.dec is not None):
```

Comment:
Pleas, see my suggestion above about removing `self.ra` and `self.dec`, they are not really doing anything as the configuration does not have these fields.